classes.py

```diff
@@ -5,6 +5,10 @@
 
 """
 from dataclasses import dataclass
+import sqlite3
+import os
+
+db = os.path.join('database', 'playlists.db')
```

Comment:
Change this so that it creates the directory and file if it doesn't already exist

Reply:
```if not os.path.exists('database'):
    os.makedirs('database')
file = 'weatherMood_library.db'
db = os.path.join('database', file)```
I've deleted the database folder and files many times and with this code the folder will be created if it doesn't exist. The file is only created after attempting to store it in the file that doesn't exist.